core/debugger/remote_debugger.cpp

```diff
@@ -38,6 +38,7 @@
 #include "core/input/input.h"
 #include "core/object/script_language.h"
 #include "core/os/os.h"
+#include "scene/main/node.h"
```

Comment:
That looks like a rebase mistake, this was actually removed on purpose in 65837973051e2aea43019fb5c20ec4a1debf0bcb. `core` shouldn't depend on `scene`.